packages/shared/src/hooks/useReadingRank.ts

```diff
@@ -63,6 +77,13 @@ export default function useReadingRank(): ReturnType {
       neverShowRankModal: newNeverShowRankModal,
     });
 
+  const shouldShowRankModal =
+    !neverShowRankModal ??
+    checkShouldShowRankModal(
+      alerts.rankLastSeen,
+      cachedRank?.rank?.lastReadTime || remoteRank?.rank?.lastReadTime,
+    );
```

Comment:
Can we not add the neverShowRankModal in the function call as well?
Just to have a sole function evaluate wether to show it.

Reply:
Great change for readability 🚀